harness/determined/pytorch/_pytorch_context.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Optional, cast
+
+import torch
+import torch.nn as nn
+
+import determined as det
+from determined import pytorch
+from determined_common import check
+
+
+class PyTorchTrialContext(det.TrialContext):
```

Comment:
Do we need a `PytorchClientContext`?

Reply:
We do not have a native API for pytorch yet, so nope.